python/ray/_private/utils.py

```diff
@@ -1893,3 +1893,25 @@ def try_import_each_module(module_names_to_import: List[str]) -> None:
             importlib.import_module(module_to_preload)
         except ImportError:
             logger.exception(f'Failed to preload the module "{module_to_preload}"')
+
+
+def update_envs(env_vars: Dict[str, str]):
+    """
+    When updating the environment variable, if there is ${X},
+    it will be replaced with the current environment variable.
+    """
+    if not env_vars:
+        return
+
+    replaceable_keys = [
```

Comment:
Can you also update the doc in https://docs.ray.io/en/master/ray-core/handling-dependencies.html#api-reference to introduce this new behavior?